ionic2/app/theme/app.md.scss

```diff
@@ -35,3 +35,6 @@ navbar .searchbar-input {
   box-shadow: inset 0 0 4px 0 rgba(0,0,0,0.2);
   border: 1px solid #b1b1b1;
 }
```

Comment:
I would like an empty line here, but dunno if this is a good coding style from my side :)